### ISSUE TITLE
[Security] Documentation still recommends the use of SHA1

### ISSUE THREAD

**Issue by ichdasich** [bigbluebutton/bigbluebutton.github.io#145]
At the moment the documentation still recommends the use of SHA1, see: https://docs.bigbluebutton.org/dev/api.html#usage

Given the security implications of SHA1, see https://github.com/bigbluebutton/bigbluebutton/issues/9634, this should be updated to recommend SHA256.

**Comment by ffdixon** [comment]
Agreed -- we do support SHA256.  Could you possibly send us a pull request to update?

**Comment by ichdasich** [comment]
Hm, digging further into this there are some more implications. Might drop you an email later.

Also, before updating the docs it might be good to also address https://github.com/blindsidenetworks/scalelite/issues/221 ; Otherwise there would be an inconsistency, and tools following the docs running against scalelight would break.
